stubs/croniter/croniter/croniter.pyi

```diff
@@ -32,32 +36,44 @@ class CroniterNotAlphaError(CroniterError): ...
 def datetime_to_timestamp(d: datetime.datetime) -> float: ...
 
 class croniter(Iterator[Any]):
-    MONTHS_IN_YEAR: Literal[12]
-    RANGES: tuple[tuple[int, int], ...]
-    DAYS: tuple[
-        Literal[31],
-        Literal[28],
-        Literal[31],
-        Literal[30],
-        Literal[31],
-        Literal[30],
-        Literal[31],
-        Literal[31],
-        Literal[30],
-        Literal[31],
-        Literal[30],
-        Literal[31],
+    MONTHS_IN_YEAR: Final[Literal[12]]
+    RANGES: Final[tuple[tuple[int, int], tuple[int, int], tuple[int, int], tuple[int, int], tuple[int, int], tuple[int, int]]]
+    DAYS: Final[
+        tuple[
+            Literal[31],
+            Literal[28],
+            Literal[31],
+            Literal[30],
+            Literal[31],
+            Literal[30],
+            Literal[31],
+            Literal[31],
+            Literal[30],
+            Literal[31],
+            Literal[30],
+            Literal[31],
+        ]
```

Comment:
Honestly, I'd be inclined to go for a simpler type here (but I don't feel strongly about it!):

```suggestion
    DAYS: Final[tuple[Literal[28, 30, 31], ...]]
```

Reply:
I like having a fixed-size tuple. On the other hand, I don't think the concrete numbers are terribly important:

```python
DAYS: Final[tuple[int, int, int, int, int, int, int, int, int, int, int, int]]
```

Reply:
Let's just leave it as it is, then -- I quite like the concrete numbers, since we know for a certainty that no month has a number of days that's not `28`, `30` or `31`, and it feels nice to me to reflect that in the stubs 😄